### PR TITLE
Update README.md Dependencies list

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -181,10 +181,10 @@ library archives (`.a`).
 | libminiupnpc | 2.0           | YES      | `libminiupnpc-dev` | `miniupnpc`  | `miniupnpc-devel` | YES      | NAT punching   |
 | libunwind    | any           | NO       | `libunwind8-dev`   | `libunwind`  | `libunwind-devel` | YES      | Stack traces   |
 | liblzma      | any           | NO       | `liblzma-dev`      | `xz`         | `xz-devel`        | YES      | For libunwind  |
-| libreadline  | 6.3.0         | NO       | `libreadline6-dev` | `readline`   | `readline-devel`  | YES      | Input editing  |
+| libreadline  | 6.3.0         | NO       | `libreadline-dev`  | `readline`   | `readline-devel`  | YES      | Input editing  |
 | ldns         | 1.6.17        | NO       | `libldns-dev`      | `ldns`       | `ldns-devel`      | YES      | SSL toolkit    |
 | expat        | 1.1           | NO       | `libexpat1-dev`    | `expat`      | `expat-devel`     | YES      | XML parsing    |
-| GTest        | 1.5           | YES      | `libgtest-dev`^    | `gtest`      | `gtest-devel`     | YES      | Test suite     |
+| GTest        | 1.5           | YES      | `libgtest-dev`     | `gtest`      | `gtest-devel`     | YES      | Test suite     |
```
Comment (moneromooo-monero) [inline README.md]: Why ?

Comment (Jorropo) [reply README.md]: Cause the ^ don't have any reason to exists.

Comment (moneromooo-monero) [reply README.md]: Why ? Is the note below obsolete ? If so, it should be removed as well. If not, it should stay unless there's a good reason for it.

Comment (Jorropo) [reply README.md]: No it was a typing error I think, the real lib name is libgtest-dev

Comment (moneromooo-monero) [reply README.md]: Sure, but the ^ refers to the note below, it's not part of the name.

**Comment by Jorropo** [comment]
After investigation except the ^ the dependencies list isn't not false, the package name effectively change a little bit between ubuntu 17.10, 18.04 and some various classic debian.

**Pull request closed without merging**